components/clarinet-files/src/network_manifest.rs

```diff
@@ -479,6 +480,10 @@ impl NetworkManifest {
                         Some(Value::Integer(balance)) => *balance as u64,
                         _ => 0,
                     };
+                    let sbtc_balance = match account_settings.get("sbtc_balance") {
+                        Some(Value::Integer(balance)) => *balance as u64,
+                        _ => 1_000_000_000, // mint 10 sBTC by default
```

Comment:
For backward compat with older generated projects, let's default to 10sBTC instead of not minting anyt sBTC if the setting file hasn't been updated